### ISSUE TITLE
TypeError: argument of type 'int' is not iterable

### ISSUE THREAD

**Issue by ghost** [ultralytics/yolov5#2810]
## 🐛 Bug
When I run Detection.py with the webcam, I get a TypeError. The error is below.

I didn't have this problem before the Youtube update. Could there be a mistake?

detect.py runs on a Nividia Jetson Nano B01

## To Reproduce (REQUIRED)

Input:
```
python3 detect.py --source 0
```
```
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', nosave=False, project='runs/detect', save_conf=False, save_txt=False, source='0', update=False, view_img=False, weights='yolov5s.pt')
YOLOv5 🚀 v5.0-9-ge5d7122 torch 1.8.1 CPU

Fusing layers... 
```

Output:
```
...
OpenBLAS Warning : Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option.
OpenBLAS Warning : Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option.
OpenBLAS Warning : Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option.
Model Summary: 224 layers, 7266973 parameters, 0 gradients, 17.0 GFLOPS
1/1: 0... Traceback (most recent call last):
  File "detect.py", line 178, in <module>
    detect()
  File "detect.py", line 51, in detect
    dataset = LoadStreams(source, img_size=imgsz, stride=stride)
  File "/home/nvidia/yolov5/utils/datasets.py", line 279, in __init__
    if 'youtube.com/' in url or 'youtu.be/' in url:  # if source is YouTube video
TypeError: argument of type 'int' is not iterable

```

## Environment

Nvidia Jetson Nano B01

**Comment by Saviourchrist** [comment]
Does anyone have a solution to the same problem

**Comment by glenn-jocher** [comment]
@cyberFoxi @Saviourchrist thank you for the bug report! We are able to correctly reproduce this error, and yes you are correct his was introduced by YouTube Livestream Detection PR #2752.

We have pushed a fix for this in YouTube Bug Fix #2818. Please git pull or clone again to receive this update and let us know if you run into any other problems!
